components/DataRequirementsViewer.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { useRecoilState } from 'recoil';
+import { selectedMeasureState } from '../atoms/selectedMeasure';
+import { Loader, ScrollArea, Card, Center, Tabs, Title, Text } from '@mantine/core';
+import dynamic from 'next/dynamic';
+const DynamicReactJson = dynamic(import('react-json-view'), { ssr: false });
+
+/**
+ * Pings DEQM-test-server for the $data-requirements results based on the measure
+ * selected. Updates each time a new measure is selected
+ * @returns JSX for Data Requirements panel on screen
+ */
+export default function DataRequirementsViewer() {
+  const [selectedMeasure] = useRecoilState(selectedMeasureState);
```

Comment:
Recoil is nice in the sense that, if you only need to read a value (rather than read/write or just write), you can just grab the value:

```suggestion
  const selectedMeasure = useRecoilValue(selectedMeasureState);
```

I prefer this syntax because it forces you to be explicit about what you want to do with the state, and doesn't have you declare an array when it isn't necessary